api/src/utils/user-utils.ts

```diff
@@ -46,12 +50,15 @@ const getViewableFields = (
 const getEditableFields = (
   currentUser: SessionUser,
   userId: string,
-): string[] => {
-  const editableFields = difference(allFields, nonEditableFields);
+): (keyof IUser)[] => {
+  const editableFields = difference(
+    allFields,
+    nonEditableFields,
+  ) as (keyof IUser)[];
   if (isAdmin(currentUser)) {
     return editableFields;
   } else if (currentUser._id.toString() === userId) {
-    return difference(editableFields, adminEditableFields);
+    return difference(editableFields, adminEditableFields) as (keyof IUser)[];
```

Comment:
lodash <3

Reply:
it will be coming to the frontend soon too